Jenkinsfile.release

```diff
@@ -51,6 +51,7 @@ ansiColor('xterm') {
         }
       } finally {
         junit(allowEmptyResults: true, testResults: 'target/test-reports/*.xml')
+        junit(allowEmptyResults: true, testResults: 'tests/integration/target/test-reports/*.xml')
```

Comment:
Hm, I'm not sure the folder structure is quite right. It seems we mix them a lot. E.g. `tests` is in some language the path what `src/test` is to JVM languages. It would only include the test sources but not the test artifacts. The same way we would not have `src/test/target/junit.xml`. We could tread `tests` as a folder for projects. However, then I would expect `tests/integration/src/main/scala`, `tests/integration/target` and probably `tests/integration/build.sbt`.

Reply:
Hmm... `src/integration/scala` ?

Reply:
They have to have separate target folders though, I think, otherwise I am not sure that running the integration tests will run the unit tests if the unit test artifacts are compiled and output in the same folder

Reply:
I like `src/integration/scala` better.